test/test.js

```diff
@@ -339,6 +339,13 @@ describe('contentDisposition.parse(string)', function () {
       })
     })
 
+    it('should parse UTF-8 extended parameter value (encoding noted "utf8")', function () {
```

Comment:
I would keep with the existing format: `should parse {charset} extended parameter value`